client/src/actions/index.js

```diff
@@ -2,7 +2,8 @@ import axios from 'axios';
 import config from '../config';
 
 // Set Up Back End URL: Change config for deployment or switch to ENV
-const url = config.backend || 'http://localhost:5000';
+// process.env.server set to heroku deployment root, already set on deployed version.
+const url = process.env.server || config.backend || 'http://localhost:5000';
```

Comment:
I am hesitant about leaving the or for localhost. It caused an issue that was hard to debug. If this runs locally and errors because there wasn't an or statement, at least we can debug it directly.